### PR TITLE
Enable Track One EventHubs Packages

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Add Track 1 EventHubs Packages to the list of Artifacts to operate on during CI runs

### CODE REVIEW

File: sdk/eventhub/ci.yml [hunk 1]
```diff
@@ -42,3 +42,7 @@ stages:
       safeName: AzureMessagingEventHubs
     - name: Azure.Messaging.EventHubs.Processor
       safeName: AzureMessagingEventHubsProcessor
+    - name: Microsoft.Azure.Management.EventHub
+      safeName: Microsoft.Azure.Management.EventHub
```
Comment (serkantkaraca) [inline sdk/eventhub/ci.yml]: Not this one. Actually these 3.

Microsoft.Azure.EventHubs
Microsoft.Azure.EventHubs.Processor
Microsoft.Azure.EventHubs.ServiceFabricProcessor

Comment (chidozieononiwu) [reply sdk/eventhub/ci.yml]: Right, fixed. The pipeline still has an issue with these, but am looking into it

**Review by serkantkaraca — CHANGES_REQUESTED** [review]
:clock1:

**Review by serkantkaraca — APPROVED** [review]
:shipit:

**Comment by serkantkaraca** [comment]
Thx for quick fix!

**Pull request merged**